backend/app/controllers/eventsController/socket/onDisconnect.js

```diff
@@ -1,4 +1,4 @@
 module.exports = (socket) => {
-    console.log("Client disconnected");
-  };
+	console.log('Client disconnected');
+};
   
\ No newline at end of file
```

Comment:
Based on the provided code patch, there do not appear to be any significant bug risks or improvement suggestions.

The only change made to the code is the adjustment of indentation and spacing for the log message. The function simply logs a message when a client disconnects. 

However, it's worth noting that the line "+\n" (no newline at end of file) could cause issues with some systems that expect a newline character at the end of files. Including a newline character after the closing curly brace could help prevent these issues.